integration_tests/test_commit_format.py

```diff
@@ -23,7 +32,15 @@ def test_commit_format():
     """
     # Fetch the upstream repository.
     fetch_base_cmd = "git fetch {} {}".format(BASE_REPO, BASE_BRANCH)
-    subprocess.run(fetch_base_cmd, shell=True, check=True)
+    try:
+        subprocess.run(fetch_base_cmd, shell=True, check=True)
+    except subprocess.CalledProcessError:
+        print(
```

Comment:
Should we use re-raise a more appropriate Exception instead? There are some details here about the best practices: https://stackoverflow.com/questions/2052390/manually-raising-throwing-an-exception-in-python

Reply:
Done.